pact/namespaces/ns.repl

```diff
@@ -1,16 +1,34 @@
 (env-exec-config ["DisablePact44", "DisablePact45"])
 (begin-tx)
+
 (env-data
  { 'ns-admin-keyset: ["admin"]
  , 'ns-operate-keyset: ["operate"]
  , 'ns-genesis-keyset: { "keys": [], "pred": "="} })
 
+(load "v1/ns.pact")
+
+(commit-tx)
+(begin-tx)
+
+;; Test that we can upgrade ns.pact only when appropriate keys
+;; are in scope.
+(env-data {})
+(expect-failure
+  "Cannot upgrade the namespace contract due to governance failure"
+  "tx failure on upgrade"
+  (try (enforce false "tx failure on upgrade") (load "ns.pact")))
+
+(env-data
```

Comment:
I think we don't need this?

Reply:
Removed the `env-data` calls here - they were unnecessary